src/app/attendance/components/AttendanceActionRender.tsx

```diff
@@ -8,31 +8,27 @@ import {
 } from "lucide-react";
 import { AttendanceStatus, MemberRole } from "@/constant/constant";
 import { useRouter } from "next/navigation";
+import { Routes } from "@/lib/routes";
 import { deleteAttendance } from "../actions/deleteAttendance";
 import { toast } from "@/hooks/use-toast";
+import { useUser } from "@/hooks/useGetLoggedinUser";
 
-interface MemberInfo {
-  shift: string;
-  role: string;
-  status: string;
-  id: number;
-}
 interface AttendanceData {
   shift: string;
   status: string;
   id: number;
 }
 
 type Props = {
-  loginUser: MemberInfo;
   attendanceData: AttendanceData;
```

Comment:
Let's just call it `attendance`. I'm not a fan of `data` and `list` as variable names.

Reply:
Fixed